plugins/experimental/stream_editor/stream_editor.cc

```diff
@@ -223,7 +223,6 @@ class rxscope : public scope_t
     if (error) {
       TSError("stream-editor: can't compile regexp [%s]", str);
       TSfree(str);
-      throw;
```

Comment:
It's not correct to simply remove these throws.  The are not rethrows, so they should throw something.  For example:
```
throw std::runtime_error("Call to regcomp() in rxscope constructor failed");
```
Continuing on with a corrupt object is unlikely to avoid a crash.  It probably will result in a crash later on, and the reason for the crash will be much, much harder to figure out.  If you don't want TS to exit due to these errors, you should add try/catch statements where ever these constructors are called, with recovery logic.

Reply:
thanks Ill give that a try, originally I was trying to use std::exception but hit some build errors with that, but Ill find  a proper exception

Reply:
runtime_error seems like an acceptable option:  https://en.cppreference.com/w/cpp/error/runtime_error